mysql-test/suite/rpl/t/rpl_percona_userstat.test

```diff
@@ -0,0 +1,60 @@
+--echo #
+--echo # Bug 995624 RBR events are not reflected in userstat's
+--echo # Rows_updated
+--echo #
+
+--source include/master-slave.inc
```

Comment:
Should go above the echo block (this is not specific to this particular bug, but is global to the test file)